draft-ietf-dnsop-rfc7958bis.mkd

```diff
@@ -419,6 +421,16 @@ This document aims to specify carefully the means by which such trust
 anchors are published, with the goal of making it easier for those
 trust anchors to be integrated into user environments.
 
+# IANA Considerations
+
```

Comment:
I think we need some comment in here that describes how this section and this document relates to the DPS that IANA publishes for management of the root zone KSK. If we need the DPS to be updated to reflect the directions in this section, for example, we should say that.

Reply:
Good call. I'll work something up for the draft, and it will get modified during WG Last Call.

Reply:
Added in 97b1b8b